### PR TITLE
Added option to ask google for directions

### LINKED ISSUE #50: Avoid walking on water
Currently the bot walks on water which can be problematic.

Ideally the bot should avoid waterways by using Google Maps directions API to navigate to the correct location.

### PR DESCRIPTION
**Pull request by andythorne**
### Short Description:

Added option to ask google for directions
### Changes:
- Added `google_directions` (boolean) config key. If set, the stepper will ask google directions API for the way (via walking) to the destination.
- Reduced long/lat randomness. each step could be altered by 9 feet (of factor 25), which sometimes sent you backwards. Now randoms to 3.64 feet (factor 10) by default.
- Closes #50 

Example in action:
map

@OpenPoGo/maintainers

### CODE REVIEW

**Comment by meyer9** [comment]
Wow! Thanks for the commits! Make sure CI passes and then we'll get this merged.

**Comment by epicblood** [comment]
could you also add this as a cli option? 
and add it to the README

**Comment by andythorne** [comment]
Updated and passing!

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/108/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

File: README.md [hunk 1]
```diff
@@ -77,9 +78,10 @@ pip install -r requirements.txt
       -cp COMBAT_POWER, --combat-power COMBAT_POWER Transfer Pokemon that have CP less than this value (default 100)",
       -iv IV, --pokemon-potential IV                Set the ratio for the IV values to transfer (DEFAULT 0.4 eg. 0.4 will transfer a pokemon with IV 0.3)
       -ri, --recycle-items                          Recycle unneeded items automatically
-      -if LIST, --item-filter LIST                  Pass a list of unwanted items to recycle when collected at a Pokestop (e.g, [\"101\",\"102\",\"103\",\"104\"] to recycle potions when collected). Requires --recycle-items. 
+      -if LIST, --item-filter LIST                  Pass a list of unwanted items to recycle when collected at a Pokestop (e.g, [\"101\",\"102\",\"103\",\"104\"] to recycle potions when collected). Requires --recycle-items.
```
Comment (wchill) [inline README.md]: Should be removed

Comment (andythorne) [inline README.md]: Done. That file needs to be cleaned up tho - lots of trailing whitespace all over it.

**Comment by MaxLeiter** [comment]
Please modify the requirements.txt to include the googlemaps API
Thanks!

**Comment by andythorne** [comment]
@MaxLeiter It's already there: https://github.com/andythorne/OpenPoGoBot/blob/dd001054aaee73fd72f6a8843b4edf5be6f1f0fd/requirements.txt#L8

**Comment by MaxLeiter** [comment]
ah, thanks for pointing that out.

:+1: great work!

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/108/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by TCB13** [comment]
What about a merge to dev?

**Comment by MaxLeiter** [comment]
We don't have a dev branch, @TCB13. This is a different project

**Comment by TCB13** [comment]
@MaxLeiter sorry, github on iPhone, never again!

**Pull request merged**